composer.json

```diff
@@ -6,22 +6,28 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.3",
-        "drupal/core": "^8.9 || ^9.1",
+        "php": ">=7.4",
+        "drupal/core": "^9.2",
         "drupal/entity_reference_revisions": "^1.6",
         "drupal/inline_entity_form": "^1.0"
     },
     "require-dev": {
+        "amphp/parallel-functions": "~1.0.0",
         "composer/installers": "~1.5",
-        "drupal/core-composer-scaffold": "^8.9 || ^9.1",
+        "drupal/core-composer-scaffold": "^9.2",
         "drupal/config_devel": "^1.2",
+        "drupal/core-dev": "^9.2",
         "drupal/drupal-extension": "^4.0",
         "drush/drush": "^10.3",
-        "openeuropa/code-review": "^1.6",
-        "openeuropa/drupal-core-require-dev": "^8.9 || ^9.1",
-        "openeuropa/task-runner-drupal-project-symlink": "^1.0",
+        "egulias/email-validator": "^2.1.22 || ^3.0",
+        "openeuropa/code-review": "^2.0",
+        "openeuropa/task-runner-drupal-project-symlink": "^1.0.0-beta5",
         "phpspec/prophecy-phpunit": "^1 || ^2"
     },
+    "_readme": [
+        "Explicit requirement amphp/parallel-functions due to grumphp errors. Remove when openeuropa/code-review will be updated",
```

Comment:
It has been soft locked in code-review 2.0.0-alpha3, so we can remove it from here.